app/src/main/java/com/tvtracker/ui/main/BrowseViewModel.kt

```diff
@@ -81,6 +79,8 @@ class BrowseViewModel(var mediaService: IMediaService = MediaService()): ViewMod
         scrollPosition = position
     }
 
+    fun getPageSize() = PAGE_SIZE
+
```

Comment:
This function just return the page size, I created this function because the page size was required in main activity.